birdhouse/env.local.example

```diff
@@ -251,7 +235,33 @@ export POSTGRES_MAGPIE_PASSWORD=postgres-qwerty
 #                                   }
 #"
 
+# Load jobs to automatically deploy the custom notebooks from the specific images
+#
+# Ensure we always use the "latest" version of the "cronjob generation code"
+# Path to a checked out repo of "pavics-jupyter-base" which contains the config required for the cronjob generation
+#CHECKOUT_PAVICS_CONFIG="/path/to/checkout/pavics-jupyter-base"
+#export AUTODEPLOY_EXTRA_REPOS="$AUTODEPLOY_EXTRA_REPOS $CHECKOUT_PAVICS_CONFIG"
+
+# Script for automatic cronjob generation
+#DEPLOY_DATA_SPECIFIC_IMAGE_JOB_ENV="$COMPOSE_DIR/components/scheduler/deploy_data_specific_image_job.env"
+# Config for the generation of cronjobs, found on external repo
+#DEPLOY_DATA_PAVICS_JUPYTER_ENV="$CHECKOUT_PAVICS_CONFIG/scheduler-jobs/deploy_data_pavics_jupyter.env"
```

Comment:
Oh so there is an override, not sure why you can not merge this override with the local `deploy_data_specific_image_job.env` but I haven't seen the override yet.